.github/workflows/CI.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           version: ${{ matrix.julia-version }}
 
       - name: Add package versions
-        run: julia --project=. -e 'using Pkg; pkg"add MakieGallery#master"'
+        run: julia --project=. -e 'using Pkg; pkg"add MakieGallery#master AbstractPlotting#master"'
```

Comment:
why `AbstractPlotting#master`? Should it not just be tested with the version specified in the Project.toml?

Reply:
There are some fixes for PlotUtils on master, which I wanted to test with CairoMakie.  I'll revert before merging.